drone/src/database.rs

```diff
@@ -132,27 +138,28 @@ impl DroneDatabase {
             subdomain
         )
         .fetch_optional(&self.pool)
-        .await?
-        .map(|d| d.address))
+        .await
     }
 
     pub async fn insert_proxy_route(
         &self,
         backend: &BackendId,
         subdomain: &str,
         address: &str,
+        bearer_token: Option<&str>,
     ) -> Result<()> {
         let backend_id = backend.id().to_string();
         sqlx::query!(
             r"
             insert or replace into route
-            (backend, subdomain, address, last_active)
+            (backend, subdomain, address, last_active, bearer_token)
```

Comment:
should we hash before storing in the db?

Reply:
Yeah, no harm in hashing these, good call.